src/styles/main.scss

```
@@ -1,3 +1,117 @@
-body {
-  margin: 0;
+@import "variables";
+@import "stars";
+@import "header";
+@import "nav";
+@import "reset";
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  background: $background--color--card;
+  border: $border--thickness solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
```

```suggestion
  padding: 32px 16px 16px;
```
and remove same margins for image and buy button